src/domain/help/layout/HelpPageLayout.tsx

```diff
@@ -101,62 +100,80 @@ const HelpPageLayout: React.FC<Props> = ({ children }) => {
       label: t('helpPage.sideNavigation.labelInstructions'),
       subLevels: instructionsSubLevels,
       to: getLocalePath(ROUTES.INSTRUCTIONS),
-      type: 'toggle',
     },
     {
       icon: <IconCogwheel />,
       label: t('helpPage.sideNavigation.labelTechnology'),
       subLevels: technologySubLevels,
       to: getLocalePath(ROUTES.TECHNOLOGY),
-      type: 'toggle',
     },
     {
       icon: <IconQuestionCircle />,
       label: t('helpPage.sideNavigation.labelSupport'),
       subLevels: supportSubLevels,
       to: getLocalePath(ROUTES.SUPPORT),
-      type: 'toggle',
     },
     {
       icon: <IconLayers />,
       label: t('helpPage.sideNavigation.labelFeatures'),
       subLevels: [],
       to: getLocalePath(ROUTES.FEATURES),
-      type: 'link',
     },
   ];
 
+  const handleLinkClick =
+    (href: string) =>
+    (
+      e:
+        | React.MouseEvent<HTMLAnchorElement, MouseEvent>
+        | React.MouseEvent<HTMLButtonElement, MouseEvent>
+        | React.KeyboardEvent<HTMLAnchorElement>
+        | React.KeyboardEvent<HTMLButtonElement>
+    ) => {
+      e.preventDefault();
+      history.push(href);
+    };
```

Comment:
Not sure if this type needs to be so strict / sepcific, maybe the following would be enough:

```
 const handleLinkClick =
    (href: string) => (e: React.MouseEvent | React.KeyboardEvent) => {
      e.preventDefault();
      history.push(href);
    };
```

Reply:
Or because the callback func is so simple, this could be inlined:

```
onClick={
                          subLevels.length
                            ? undefined
                            : (e) => {
                                e.preventDefault();
                                history.push(to);
                              }
                        }
```

Reply:
Same function is used twice in this component so decided to use the first suggestion